apps/orams/components/BusinessInfoForm/BusinessInfoForm.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable */
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Form, Control } from 'react-redux-form'
+
+import Layout from 'shared/Layout'
+import BaseForm from 'shared/form/BaseForm'
+import SubmitForm from 'shared/form/SubmitForm'
+import ErrorBox from 'shared/form/ErrorBox'
+import StatefulError from 'shared/form/StatefulError'
+import { required } from 'shared/validators'
+import formProps from 'shared/form/formPropsSelector'
+import { loadProfile } from 'orams/actions/profileActions'
+
+class BusinessInfoForm extends BaseForm {
+  static propTypes = {
+    action: PropTypes.string,
+    csrf_token: PropTypes.string,
+    form: PropTypes.object.isRequired,
+    returnLink: PropTypes.string
+  }
+
+  componentDidMount() {
+    this.props.loadProfileData(this.props.form.model)
```

Comment:
when I've used the load initial data action, it was in `componentWillMount`, wonder if it makes a difference?